test/libsolidity/lsp/include-paths/default-include.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.0;
+
+import "my-module/test.sol";
```

Comment:
Can you add a test that imports just `test.sol`?

Reply:
We have general import tests already for import path names that are meant to be relatively imported. What do you think should we expect of your import-test.sol-testcase?

Reply:
I was expecting it to trigger the problem I mentioned in https://github.com/ethereum/solidity/pull/12994#discussion_r874118206.